react/features/calendar-sync/web/googleCalendar.js

```diff
@@ -62,5 +63,18 @@ export const googleCalendarApi = {
      */
     _isSignedIn() {
         return () => googleApi.isSignedIn();
+    },
+
+    /**
+     * Updates calendar event by generating new invite URL and editing the event
+     * adding some descriptive text and location.
+     *
+     * @param {string} id - The event id.
+     * @param {string} calendarId - The id of the calendar to use.
+     * @param {string} location - The location to save to the event.
+     * @returns {function(Dispatch<*>): Promise<string|never>}
+     */
+    updateCalendarEvent(id: string, calendarId: string, location: string) {
```

Comment:
If you prefer this could just be `updateCalendarEvent`, without having to re-declare the signature or calling it again below.